test/spec.yml

```diff
@@ -91,6 +92,12 @@
         out: Hello unguessable!
     -   in: bn invoke extrasmallaunt -j invoke.json
         out: Hello unguessable!
+    -   in: bn create alwaysucceeds
```

Comment:
I'm not sure what sort of coverage this really adds. e.g. it doesn't really check that the version being invoked is the version that was deployed, right?

Reply:
Well technically since it's deployed and invoked only once and then removed it should be a guarantee. That being said I'll make the output/message unique to avoid confusion.